### DISCUSSION TITLE
Not Received GitHub Education Benefits Approved on October 8th

### DISCUSSION THREAD

**Discussion by syed-kifayat** [community/community#141161] [GitHub Education]
### Hi everyone,

Hello everyone,

I recently received approval for my GitHub Education benefits on October 8th, but I have yet to receive any of the benefits. I’ve checked my email and spam folder but haven’t found any communication regarding this.

Has anyone else experienced a similar delay? If so, how long did it take for your benefits to be processed? Any advice on how to follow up or resolve this issue would be greatly appreciated.

Thank you!

**Comment by davevad93** [comment]
Hi @syed-kifayat ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @syed-kifayat was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
